Assets/Editor/ModelInspector.cs

```diff
@@ -178,8 +190,21 @@ void CheckAvailableAttributes()
                     m_Settings.availableUVChannels[index] = false;
                 index++;
             }
+
+            if (mesh.blendShapeCount > 0)
```

Comment:
mesh.blendShapeCount calls into C++ code each time it's used, so here it does a bunch of calls in the loop for no good reason. Query that once into a local variable.